bin/fby_client

```diff
@@ -40,57 +24,60 @@ def get_sys_info():
 
 class FbyRunner(object):
 
-    def __init__(self, var_path=None):
+    def __init__(self, root=None, config_file=None, var_path=None):
+        if not root:
```

Comment:
Yes, thanks for the feedback.  I will add a short comment, but I agree with what you have mentioned earlier;  we should go over to **XDG**;  then `etc` would be replaced with `$XDG_CONFIG_DIRS`, `var` (or data) with `$XDG_DATA_DIRS` and `bin` will probably be something like `/usr/local/bin`.

So, we put
* `friskby.sql` into `$XDG_DATA_DIRS/friskby` (e.g. `/usr/local/share/friskby/friskby.sql`)
* `friskby.service` into `$XDG_DATA_DIRS/friskby` (e.g. `/usr/local/share/friskby/friskby.service`)
* `config.json` into `$XDG_CONFIG_DIRS/friskby` (e.g. `/etc/xdg/friskby/config.json`)